web-app/js/portal/details/SpatialSubsetControlsPanel.js

```diff
@@ -40,43 +38,52 @@ Portal.details.SpatialSubsetControlsPanel = Ext.extend(Ext.Panel, {
             trackFiltersUsage('filtersTrackingSpatialConstraintAction', OpenLayers.i18n('trackingValueCleared'));
         }, this);
 
-        var spacer = new Ext.Spacer({
-            width: 10
-        });
-
         var pickerPanel = new Ext.Panel({
-            layout: 'hbox',
+            width: 135,
+            cls: "floatLeft",
             items: [
+                this._addLabel(OpenLayers.i18n('spatialExtentHeading')),
+                this._addVerticalSpacer(10),
```

Comment:
Just a thought, what do you think about making the heights and widths constants here?